PCbuild/get_external.py

```diff
@@ -3,23 +3,47 @@
 import argparse
 import os
 import pathlib
+import random
 import sys
 import time
 import zipfile
 from urllib.request import urlretrieve
 
 
+def retrieve_with_retries(download_location,
+                          output_path,
+                          reporthook,
+                          max_retries = 7,
+                          base_delay = 2.25,
+                          max_jitter = 1.0):
```

Comment:
Let's use more common wrap style:
```suggestion
def retrieve_with_retries(download_location, output_path, reporthook,
                          max_retries=7, base_delay=2.25, max_jitter=1.0):
```